content/blog/kubecon-2019/index.md

```diff
@@ -0,0 +1,271 @@
+---
+title: "KubeCon NA 2019: Are you about to break Prod?"
+date: 2019-11-26
+meta_desc: "In this post, we recap and add more details to our KubeCon talk on setting up ephemeral environments. We will highlight how to acceptance test your application and infrastructure using ephemeral environments."
+meta_image: demo-arch.jpg
+authors: ["erin-krengel"]
+tags: ["testing", "ephemeral environments", "KubeCon", "kubernetes"]
+---
+
+A couple of weeks ago, my colleague Sean and I had the opportunity to present "Are you about to break prod? Acceptance Testing with Ephemeral Environments" at KubeCon NA 2019. In this talk, we covered what is an ephemeral environment, how to create one, and then we walked the audience through a concrete example. Given our limited time, we had to move quickly through a ton of information. Today, I will recap our presentation and add a few more details we weren't able to cover.
+
+<!--more-->
+
+As software engineers, our job is to deliver business value. To do this, we need to be delivering software both quickly and reliably.
+
+So the question we ask you is: are you about to break prod? We will all break production at some point because there are things we miss. As Alexandra points out in her tweet, "failures are part of the cost of building and shipping large systems". Building a robust pipeline, allows us to move quickly in the case of failure and gain confidence around making changes to our infrastructure and applications.
```

Comment:
```suggestion
So the question we ask you is: are you about to break prod? Everyone will break production at some point because there are things we miss. As Alexandra points out in her tweet, "failures are part of the cost of building and shipping large systems". Building a robust pipeline allows us to move quickly in the case of failure and gain confidence around making changes to our infrastructure and applications.
```